wsgi/playdoh.wsgi

```diff
@@ -9,7 +11,7 @@ except ImportError:
 
 
 if newrelic:
-    newrelic_ini = os.getenv('NEWRELIC_PYTHON_INI_FILE', False)
+    newrelic_ini = config('NEWRELIC_PYTHON_INI_FILE', default='newrelic.ini')
```

Comment:
The default was `False` before so that not defining the Env var would effectively turn NR off w/o having to uninstall the lib. Do you think that's valuable? We can leave it like this, but I also think it's fine to require the env var for NR to work.